.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+** Specifics (please complete the following information):**
+ - OS: 
+ - Browser:
```

Comment:
Let's also add the Python version here. Maybe we can also make the browser optional, since most issues here may be with the framework itself